CHANGELOG.rst

```diff
@@ -2,6 +2,36 @@
 Changelog for package kvaser_interface
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+2004.1.1 (2021-08-20)
```

Comment:
what's the meaning of 2004??

Reply:
I'm starting all ROS2 release versions at 2000+, and will try to match the major version of the ROS1 release that has similar functionality. So the 2004.1.1 is the ROS2 version of the 4.1.1 release.